apps/frontend/src/libs/components/sidebar/sidebar-item/sidebar-item.tsx

```diff
@@ -0,0 +1,30 @@
+import { Icon, Link } from "~/libs/components/components.js";
+import { type IconName } from "~/libs/components/icon/types/types.js";
+import { type AppRoute } from "~/libs/enums/enums.js";
+import { type ValueOf } from "~/libs/types/types.js";
+
+import styles from "./styles.module.css";
+
+type Properties = {
+	href: undefined | ValueOf<typeof AppRoute>;
```

Comment:
```suggestion
	href: ValueOf<typeof AppRoute>;
```
the href should be all the time